packages/flutter_tools/test/commands.shard/hermetic/doctor_test.dart

```diff
@@ -30,7 +30,7 @@ import 'package:flutter_tools/src/version.dart';
 import 'package:flutter_tools/src/vscode/vscode.dart';
 import 'package:flutter_tools/src/vscode/vscode_validator.dart';
 import 'package:flutter_tools/src/web/workflow.dart';
-import 'package:mockito/mockito.dart';
+import 'package:test/fake.dart';
 
 import '../../src/common.dart';
 import '../../src/context.dart';
```

Comment:
GitHub won't let me comment on the line, but I don't think `noColorTerminalOverride` is necessary, I removed it from your patch and the tests still passed.

Reply:
Done!